e2e/test/scenarios/actions/model-actions.cy.spec.js

```diff
@@ -714,6 +732,8 @@ describe(
 
       cy.findByRole("button", { name: "Update" }).click();
 
+      cy.wait("@updateAction");
```

Comment:
This is a classic cause of many flakes. We click a button in the UI that fires off an API request, then we expect to do something with application state that depends on that API request.

In many cases, we're just looking for data that isn't there yet. In this case, we navigate to a new page, and I think sometimes cancel some API requests, so they never even complete.

**TLDR; Always wait on API requests that you're triggering.**